graph-builder/tests/images/test-public-multiarch-manual-0.0.0/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM scratch
+COPY release-metadata release-manifests/release-metadata
```

Comment:
Do we expect these to diverge between images?  Might want to just bake this into the script from d091fac:

```sh
for target in "${@}"; do
  ...
  cat <<-EOF >"${target}/Dockerfile"
    FROM scratch
    COPY release-metadata release-manifests/release-metadata
    EOF
  ...
done
```

Or skip the Dockerfile and `buildah bud ...` altogether and do:

```sh
container="$(buildah from scratch)"
buildah copy "${container}" "${target}/release-metadata" /release-manifests/release-metadata
```

or whatever (untested).

Reply:
They are shared between a lot of images but not all of them. Some have multiple layers which are crafted to find issues with layer ordering or race-conditions.

Reply:
> Some have multiple layers which are crafted to find issues with layer ordering or race-conditions.
Can we unit-test that separately?

Reply:
I think so, yes. Let's consider that in the test refactoring we have planned and on which I'll be working on next. It's not a multi-arch specific issue, so I would prefer not to block this PR on that.